Lib/datetime.py

```diff
@@ -798,7 +864,7 @@ def __new__(cls, year, month=None, day=None):
         """
         if (month is None and
             isinstance(year, (bytes, str)) and len(year) == 4 and
-            1 <= ord(year[2:3]) <= 12):
+                1 <= ord(year[2:3]) <= 12):
```

Comment:
I'm puzzled by your autoformatter. It indented the third line and not the second? Surely if it's going to fix one it should fix both? Regardless, not a necessary part of the patch.

Reply:
https://github.com/PyCQA/pycodestyle/issues/1066